articles/intro-to-katas.md

```diff
@@ -27,47 +28,59 @@ You can use the Katas for learning in your environment of choice:
+|[Joint measurements](https://github.com/microsoft/QuantumKatas/tree/master/JointMeasurements)|Learn about joint parity measurements and how to use the [Measure](xref:microsoft.quantum.intrinsic.measure) operation to distinguish quantum states.|
 
 ### Algorithms
 
-* [Quantum teleportation](https://github.com/microsoft/QuantumKatas/tree/master/Teleportation)
-* [Superdense coding](https://github.com/microsoft/QuantumKatas/tree/master/SuperdenseCoding)
-* [Deutsch–Jozsa algorithm](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/ExploringDeutschJozsaAlgorithm)
-* [Implementing Grover's search algorithm](https://github.com/microsoft/QuantumKatas/tree/master/GroversAlgorithm)
-* [Exploring high-level properties of Grover's search algorithm](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/ExploringGroversAlgorithm)
-* Solving real problems using Grover's algorithm: [SAT problems](https://github.com/microsoft/QuantumKatas/tree/master/SolveSATWithGrover) and [graph coloring problems](https://github.com/microsoft/QuantumKatas/tree/master/GraphColoring)
+| Kata | Description |
+|:-----|-------------|
+|[Quantum teleportation](https://github.com/microsoft/QuantumKatas/tree/master/Teleportation)|This kata explores quantum teleportation - a protocol which allows communicating a quantum state using only classical communication and previously shared quantum entanglement.|
+|[Superdense coding](https://github.com/microsoft/QuantumKatas/tree/master/SuperdenseCoding)|Superdense coding is a protocol that allows transmission of two bits of classical information by sending just one qubit using previously shared quantum entanglement.  |
+|[Deutsch–Jozsa algorithm](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/ExploringDeutschJozsaAlgorithm)|This algorithm is famous for being one of the first examples of a quantum algorithm that is exponentially faster than any deterministic classical algorithm.|
+|[Exploring high-level properties of Grover's search algorithm](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/ExploringGroversAlgorithm)|A high-level introduction to one of the most famous algorithms in quantum computing. It solves the problem of finding an input to a black box (oracle) that produces a particular output. |
+|[Implementing Grover's search algorithm](https://github.com/microsoft/QuantumKatas/tree/master/GroversAlgorithm)|This kata dives deeper into Grover's search algorithm, and covers writing oracles, performing steps of the algorithm, and finally putting it all together.|
+|[SAT problems](https://github.com/microsoft/QuantumKatas/tree/master/SolveSATWithGrover)|A series of exercises that uses Grover's algorithm to solve realistic problems, using [boolean satisfiability problems](https://en.wikipedia.org/wiki/Boolean_satisfiability_problem) (SAT) as an example.  |
```

Comment:
I wouldn't use shorthand here and in the next item - we're not discussing the problems as such here, we're discussing solving them using Grover's algorithm, so a shorthand is confusing. "Solving ... problems" is a better wording, used in the katas.